src/ASM/ASMs3DmxLag.C

```diff
@@ -430,7 +429,7 @@ bool ASMs3DmxLag::integrate (Integrand& integrand, int lIndex,
     for (size_t t = 0; t < threadGrp[g].size(); t++)
     {
       MxFiniteElement fe(elem_size);
-      Matrices dNxdu;
+      BasisValues bfs(elem_size.size());
```

Comment:
this fixes a bug.

Reply:
what bug? The BasisValues object was introduced in this commit for this method, afics

Reply:
the dNxdu was empty previously...

Reply:
Indeed, which means that mixed 3D Lagrange (with Neumann conditions) is not covered by any of the regression tests.